corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -1011,8 +1027,22 @@ class ConfigureListReportForm(ConfigureNewReportBase):
 
     @property
     def container_fieldset(self):
+        source_name = ''
+        if self.source_type == 'case':
+            source_name = self.report_source_id
+        if self.source_type == 'form':
+            source_name = Form.get_form(self.report_source_id).default_name()
         return crispy.Fieldset(
-            "",
+            '',
+            crispy.Fieldset(
+                _legend(
+                    _("Rows"),
+                    _('This report will show one row for each {name} {source}'.format(
```

Comment:
string formatting should be applied after translations so the params get properly injected into the translated string instead of trying to translate the formatted version (which won't be found in translations)